ui/desktop/app/styles/app.scss

```diff
@@ -530,3 +530,111 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     background-color: var(--token-color-palette-neutral-50);
   }
 }
+
+.details-screen-body {
+  .rose-layout-body-content {
+    margin-top: 1.5rem;
+  }
+}
+
+// Session Details - Credentials Panel
+.credential-panel-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 0.5rem;
+
+  .title {
+    font-weight: var(--token-typography-font-weight-bold);
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  .hds-form-field--layout-flag {
+    margin-bottom: 0;
+  }
+}
+
+.credential-panel-body {
+  border: 0.063rem solid var(--token-color-border-primary);
+  border-radius: 0.313rem;
+  display: flex;
+  margin-top: 0.75rem;
+
+  .credential-header {
+    padding: 1.5rem;
+    width: 33%;
+  }
+}
+
+.credential-secret {
```

Comment:
None blocking comment: If `.credential-secret` just lives inside of a `.credential-panael-body` I will suggest to nested inside `.credential-panel-body` as we did with `.credential-header`. Thoughts? 

Reply:
I originally had it nested but I got a lint error because it was too many items nested.
`app/styles/app.scss
 587:9  ✖  Expected ".credential-panel-body .credential-secret .secret-container .hidden-secret .rose-button-ghost" to have no more than 4 compound selectors  selector-max-compound-selectors`

Reply:
I wouldn't disable the rule. I never had this issue so I can't talk 100% with confidence. But the rule you mention, as [per my understanding](https://stylelint.io/user-guide/rules/selector-max-compound-selectors/#:~:text=Limit%20the%20number%20of%20compound%20selectors%20in%20a%20selector.&text=A%20compound%20selector%20is%20a,%60%2C%20%2B%20%2C%20%3E%20).) is about how many selectors you use to chain, now about how many selectors are nested.

Perhaps we can do something like:
```
.credential-secret {
& > .credential-secret {}
& > .credential-heder {}
}

Reply:
Styles updated to please stylelint rule